Since linux-4.14.9 there has been some kernel headers reordering.
Fix that issue by including linux/sched/task_stack.h for kernels
newer than 4.14.9.

--- NVIDIA-Linux-x86_64-390.25-no-compat32/kernel/nvidia-uvm/uvm8_va_block.c	2017-10-27 01:19:54.000000000 +0200
+++ NVIDIA-Linux-x86_64-390.25-no-compat32/kernel/nvidia-uvm/uvm8_va_block.c	2018-01-02 02:50:05.260588964 +0100
@@ -36,6 +36,10 @@
 #include "uvm8_perf_prefetch.h"
 #include "uvm8_mem.h"
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(4,14,9)
+#include <linux/sched/task_stack.h>
+#endif
+
 typedef enum
 {
     BLOCK_PTE_OP_MAP,
